jwst/tests_nightly/general/fgs/test_fgs_image2_1.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from jwst.pipeline.calwebb_image2 import Image2Pipeline
+
+from ..resources import FGSTest
+
+
+@pytest.mark.bigdata
+class TestImage2Pipeline(FGSTest):
+    ref_loc = ['test_image2pipeline']#, 'truth']
+
+    def test_fgs_image2pipeline1(self, _jail):
```

Comment:
This is not a full review, but at a glance, I can tell you that you don't have to pass in `_jail` in every method like this. See https://github.com/spacetelescope/hstcal/blob/ed4e9e90845ae9d281fec0cb26aba41a5433dae1/tests/helpers.py#L133 and https://github.com/spacetelescope/hstcal/blob/ed4e9e90845ae9d281fec0cb26aba41a5433dae1/tests/helpers.py#L145 in case it helps.